supervisor/shared/filesystem.c

```diff
@@ -69,6 +69,23 @@ static void make_empty_file(FATFS *fatfs, const char *path) {
     f_close(&fp);
 }
 
+
+static void make_sample_code_file(FATFS *fatfs) {
+    FIL fs;
```

Comment:
These 3 variables are only used in the CIRCUITPY_FULL_BUILD branch, so move them just after `#else`.

Reply:
Done :)